src/oemof/visio/energy_system_graph.py

```diff
@@ -28,6 +29,22 @@
 # new with oemof-solph 0.5.2
 from oemof.solph.buses._bus import Bus
 
+try:
+    from oemof.network.network import SubNetwork, Node
+    from oemof.network.network import HierachicalLabel
+
+    SUBNETWORK_MODULE = True
+except:
+
+    class SubNetwork:
+        pass
+
+    SUBNETWORK_MODULE = False
+    warnings.warn(
+        "Your oemof-network version doesn't support the SubNetwork, if "
+        "you want to work with SubNetworks, please update oemof-solph"
+    )
```

Comment:
Do we need to warn here? If the users have an old version of oemof.network, they do not use `SubNetwork`s  should be fine without the warning.

Reply:
We don't need to :)

Reply:
I removed it now